atom/browser/atom_javascript_dialog_manager.cc

```diff
@@ -91,10 +91,18 @@ void AtomJavaScriptDialogManager::RunJavaScriptDialog(
       window = relay->GetNativeWindow();
   }
 
+  atom::MessageBoxSettings settings;
+  settings.parent_window = window;
+  settings.type = atom::MessageBoxType::kNone;
```

Comment:
I would use `atom::MessageBoxType::kNone` as default value in the struct instead